client/app.jsx

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
 import Header from './header';
 import NewItems from './pages/new-items';
+import parseRoute from '../server/parse-route';
```

Comment:
If you put it in your client directory, there's no reason to import it from the server directory. I recommend a `lib` directory for helper functions.